be/src/exprs/bloomfilter_predicate.h

```diff
@@ -206,11 +206,12 @@ struct StringFindOp {
 template <class BloomFilterAdaptor>
 struct FixedStringFindOp : public StringFindOp<BloomFilterAdaptor> {
     ALWAYS_INLINE bool find_olap_engine(const BloomFilterAdaptor& bloom_filter,
-                                        const void* data) const {
-        const auto* value = reinterpret_cast<const StringValue*>(data);
-        auto end_ptr = value->ptr + value->len - 1;
-        while (end_ptr > value->ptr && *end_ptr == '\0') --end_ptr;
-        return bloom_filter.test_bytes(value->ptr, end_ptr - value->ptr + 1);
```

Comment:
Why the origin implement can' t handle char(0)?

Reply:
Because the size of the empty string is 1.
It is expected that the parameter passed into the function test_bytes should be 0. 
he previous logic cannot turn 1 into 0.